### PR TITLE
ToolTip data visible in DOM if touchscreen

### PR DESCRIPTION
**Pull request by booellean**
Additional JS file has a touch event that adds a class to the div.tooltips, aka the divs that contain volunteer info. The css positions the div into the DOM and prevents hover events for these divs.

### CODE REVIEW

File: app/assets/javascripts/mobile.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+window.addEventListener('touchstart', addMobileClass);
+
+function addMobileClass(){
+  let allTooltipDivs = document.querySelectorAll('.the-tooltip');
+  let allDescriptions = document.querySelectorAll('.tooltip-description');
+
+  allTooltipDivs.forEach( (tooltip)=> {
+    // tooltip.classList.add('mobile');
```
Comment (cherchezlafemme) [inline app/assets/javascripts/mobile.js]: remove comment

**Comment by cherchezlafemme** [comment]
@booellean can you merge in master and make sure the css looks good? you may will need to remove the restriction of width: 70% once master is merged in. the tooltip carrot is centered in master.
The behaviour seems a little off. It works great, however then I click on 1 person, all the tooltips opens up. and there is no way for me to close it. What if we add toggle classes instead? Or at least do so that it opens up only the person you want to see info about.  Let me know your thoughts.

**Comment by booellean** [comment]
Hey @cherchezlafemme, We could modify individual elements to respond to touch events, but that may replace old issues with new ones. My primary concern is if a user wants to copy and paste something they read from the user or the tooltip, it will randomly open and close while tapping. On the other hand, I could try linking the event handler to the profile image which will remove those possible frustrations since the image is smaller and less likely to be accidentally touched. However, I guarantee I'm going to have to change some of the html which will probably disrupt the current css. I am confident I can fix it, it will just take a lot of time.
I'll leave it up to you. Would you like me to take some more time and implement those features?

**Comment by booellean** [comment]
@cherchezlafemme I'll get right no that!

**Comment by booellean** [comment]
@cherchezlafemme So, for toggle to work, we'll need to be able to treat the user image/div as a button-like object, so when a user taps the image/div the (currently) tootltip div will open up.

However, the image/div is a hyperlink and leads to the team members linkedin page. To have separate click events will be a UX nightmare.

Since we only want the toggle function on touchscreen devices, we can change the hyperlink behaviors on touchscreens as well by removing the hyperlink on the image and placing a hyperlink in the tooltip after the description section.

Would you like me to implement it this way or would you like it done another way?

**Comment by booellean** [comment]
@cherchezlafemme I created a pull request so you can see what I mean and play with the possible layout. All the toggle features you requested are there and I can make any necessary edits to it.

Please refer to https://github.com/shescoding/shescoding-dot-org/pull/173 .  I will close this pull request.

**Pull request closed without merging**